test/factories/host.rb

```diff
@@ -0,0 +1,15 @@
+factory :os do
```

Comment:
Can you name this file foreman_custom_tab_factories.rb, since its not only hosts.